src/nvim/runtime.c

```diff
@@ -1156,17 +1168,15 @@ void load_plugins(void)
 {
   if (p_lpl) {
     char *rtp_copy = p_rtp;
-    char *const plugin_pattern_vim = "plugin/**/*.vim";  // NOLINT
-    char *const plugin_pattern_lua = "plugin/**/*.lua";  // NOLINT
+    char *const plugin_pattern = "plugin/**/*.\\(vim\\|lua\\)";  // NOLINT
```

Comment:
```suggestion
    char *const plugin_pattern = "plugin/**/*.{vim,lua}";  // NOLINT
```

Reply:
I've updated the change so the code doesn't rely on suffix patterns and the vim and lua case now has a specialized path.